api/v1beta1/machine_types.go

```diff
@@ -56,6 +56,22 @@ const (
 	// This annotation can be set on BootstrapConfig or Machine objects. The value set on the Machine object takes precedence.
 	// This annotation can only be used on Control Plane Machines.
 	MachineCertificatesExpiryDateAnnotation = "machine.cluster.x-k8s.io/certificates-expiry"
+
+	// NodeRoleLabelPrefix is one of the CAPI managed prefix Node labels.
```

Comment:
Q: Should we add a note that those labels are reconciled only on node creation?  (same for node-restriction)

Reply:
I think we decided to always reconcile  (the one time part didn't make it into the merged proposal)